shared/modules/network.utils.ts

```diff
@@ -56,3 +57,15 @@ export function isTokenDetectionEnabledForNetwork(chainId: string | undefined) {
 function isSafeInteger(value: unknown): value is number {
   return Number.isSafeInteger(value);
 }
+
+export async function shouldShowLineaMainnet(rpcUrl: string): Promise<boolean> {
```

Comment:
Nit: if `currentDateHigherThanReleaseDate` is false, then we can return false before we make the `jsonRpcRequest`